db/migrate/20171107000347_add_ssn_to_member_and_medicaid_application.rb

```diff
@@ -0,0 +1,12 @@
+class AddSsnToMemberAndMedicaidApplication < ActiveRecord::Migration[5.1]
+  def up
+    add_column :members, :encrypted_ssn, :string
+    add_column :members, :encrypted_ssn_iv, :string
+    add_column :medicaid_applications, :encrypted_ssn, :string
```

Comment:
I don't think we need to add this at this point -- we are only saving SSN on `members` for medicaid apps.

if my memory is correct, we were keeping `last_four_ssn` around on medicaid apps because we'd collected that data on the medicaid app directly for the single member flow but never got around to transferring that data onto the primary member for legacy medicaid_applications.

Reply:
Ahhh that was it! Thanks! Just added migration to remove.